doc/user/quickstart.rst

```diff
@@ -9,63 +9,115 @@ Quickstart
 Using the CLI
 -------------
 
-Publish a message
+Publish messages
 ^^^^^^^^^^^^^^^^^
 
 .. code:: bash
 
-    hop publish kafka://hostname:port/gcn -f circular example.gcn3
+    hop publish kafka://hostname:port/gcn -f CIRCULAR example.gcn3
 
-Two example messages (an RFC 822 formatted GCN circular (:code:`example.gcn3`) and a VOEvent 2.0
-schema xml (:code: `example_voevent.xml`)) are provided in :code:`tests/data`.
+Example messages are provided in :code:`tests/data` including:
 
-Client `configuration <https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md>`_
-properties can be passed to :code:`hop publish` via :code:`-X property=value` or in a configuration
-file specified by :code:`-F <config-file>`, mimicking the behavior of :code:`kafkacat`. This can be
-used to connect to a Kafka broker with SSL authentication enabled, for example.
+* A GCN circular (:code:`example.gcn3`)
+* A VOEvent (:code:`example_voevent.xml`)
 
-Consume a message
+Consume messages
 ^^^^^^^^^^^^^^^^^
 
 .. code:: bash
 
-    hop subscribe kafka://hostname:port/gcn -e
+    hop subscribe kafka://hostname:port/gcn -s EARLIEST
 
-Configuration properties can be passed in a manner identical to :code:`hop publish` above.
+This will read messages from the gcn topic from the earliest offset
+and read messages until an end of stream (EOS) is received.
 
+By default, authentication is enabled, reading in configuration settings
+from :code:`auth.conf`. The path to this configuration can be found by running
+:code:`hop auth locate`. One can initialize this configuration with default
+settings by running :code:`hop auth setup`. To disable authentication in the CLI
+client, one can run :code:`--no-auth`.
 
-Reading messages from stream
-----------------------------
+Using the Python API
+----------------------
 
-The hop client supports a python-based API for reading messages from a stream, as follows:
+Publish messages
```

Comment:
Nice catch, I've done a little more rearranging so that the order of producing/consuming messages is the same for both the CLI and python API sections.